apps/entropy-tester/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@pythnetwork/entropy-tester",
+  "version": "1.0.0",
+  "description": "Utility to test entropy provider callbacks",
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "files": [
+    "lib/**/*"
+  ],
+  "scripts": {
+    "build": "tsc",
+    "fix:format": "prettier --write \"src/**/*.ts\"",
+    "fix:lint": "eslint src/ --fix --max-warnings 0",
+    "test:format": "prettier --check \"src/**/*.ts\"",
+    "test:lint": "eslint src/ --max-warnings 0",
+    "start": "node lib/index.js",
+    "dev": "ts-node src/index.ts",
+    "prepublishOnly": "pnpm run build && pnpm run test:lint",
+    "preversion": "pnpm run test:lint",
+    "version": "pnpm run test:format && pnpm run test:lint && git add -A src"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/pyth-network/pyth-crosschain.git",
+    "directory": "apps/entropy-tester"
+  },
+  "bin": {
+    "pyth-entropy-tester": "./lib/index.js"
+  },
+  "devDependencies": {
+    "@types/ethereum-protocol": "^1.0.2",
+    "@types/express": "^4.17.21",
+    "@types/jest": "^27.4.1",
+    "@types/yargs": "^17.0.10",
+    "@typescript-eslint/eslint-plugin": "^6.0.0",
+    "@typescript-eslint/parser": "^6.0.0",
+    "eslint": "^8.13.0",
+    "jest": "^29.7.0",
```

Comment:
In general you should use `catalog:` dependencies for anything that's listed in [the workspace catalog](https://github.com/pyth-network/pyth-crosschain/blob/main/pnpm-workspace.yaml#L48-L152).  Doing so will enable me to maintain the versions as I upgrade shared monorepo dependencies.

At a minimum I would ask that you do this for code checking tools like eslint, jest, prettier, etc.